Removes Science Outpost(s), makes Cyb's z5 generic and used by Meta and Cyberiad

FreeStylaLT opened the pull request:
Replaces it with a regular Mining Outpost with a mech recharger:

BEFORE:
![img](http://i.imgur.com/K0vsfjK.png)

AFTER:
![img](http://i.imgur.com/yzRCwq3.png)

**The dock:**

BEFORE:
![img](http://i.imgur.com/0UnrJsi.png)

AFTER:
![img](http://i.imgur.com/YVocP08.png)

**META:**

BEFORE:
![img](http://i.imgur.com/hj8mCCN.png)

AFTER:
![img](http://i.imgur.com/UWzo9VT.png)

BEFORE:
![img](http://i.imgur.com/izNHK3z.png)

AFTER:
![img](http://i.imgur.com/yzRCwq3.png)


This more-or-less had to be done eventually, as the outpost had literally no uses beyond some degree of antag-haven / emergency shelter.

Mech Recharger added to the Mining outpost due to it being the furthest possible point, to serve as a mech-checkpoint if mining the north-west point of the asteroid, but no big deal if it's deemed too much.

🆑 FreeStylaLT
del: Removed sci outpost and dock from Metastation and Cyberiad
add: Added a Mining Outpost to the northernmost point of the Asteroid.
/🆑

tigercat2000 commented:
Remove it on Metastation too, please.

TravellingMerchant commented:
About time. :+1:  Nice PR.

ProperPants commented:
Honestly, you don't even need that little 2x2 room left over from removing the shuttle. It could just be a single continuous maint tunnel now, with the purple RnD door becoming a maint door.

FreeStylaLT commented:
Also removes it from Meta, relevant pics updated.

SamHPurp commented:
@tigercat2000 with Meta, its worth me just doing it on my PR instead maybe. *nudge nudge merge?*

FreeStylaLT commented:
@SamHPurp too late

taukausanake commented:
butbutbut I wanted to NUKE the outpost! Turn it into a crater, an abandoned outpost lonely and forgotten, maybe with minor loot and stuff that acts as a reminder of a better time

Aurorablade commented:
*hiss *hiss *hiss *hiss

deep-fried-faceplant commented:
I really like this. There was only one legitimate use for the outpost, and that was evacuating people during code delta. The other reason it was used was for cult bases, and dealing with antagonists who use the science outpost as a base was incredibly challenging, especially if security isn't powergaming or using meta knowledge. There was no reason for science to keep the outpost after xenoarch was removed, and I'm glad to see the outpost finally get the chopping block.

scrubmcnoob commented:
but muh science rp

SamHPurp commented:
@FreeStylaLT reee you didn't even maintain lattice parity. I'll do this on my Meta 2.0 branch.

Rumiluntti commented:
muh cult base!
muh monkeyman succing closet!
muh corpse pile!

Taaketa commented:
I keep saying it. Rotate the Sec Outpost to where the Sci Outpost is. It makes the Sec Outpost actually useful since prisoners won't have to fight xenos on the asteroid now. Then seal up the gap in the middle of the mining asteroid so that intrepid prisoners can mine their way to Russian mining base.

AffectedArc07 commented:
Muh malf AI
Muh monkeyman cheese
Muh cult base
Muh secret bunker
Muh nukeop shelter

👎 * 💯

imsxz commented:
Muh surplus crate buying location

ktoma36 commented:
Would there be a possibility of moving the anomaly research stuff on to station z-level? Could use the free space from where the shuttle was to hold all of it. I would love to actually RESEARCH anomaly cores instead of robotics just hoovering them up for meme mechs.

Kyep commented:
👍 

RIP, science outpost. You won't be missed, except by antags that want to do cheesy things off-station.

Citinited commented:
I occasionally use the RD outpost for putting out bombs for mining, as I can never quite bring myself to trust miners with maxcaps. I suppose I'll have to go through the HoP now... (Oh god that's going to never happen is it?)

scrubmcnoob commented:
why do people care
this is going be gone anyway when we totally get lavaland

granodd commented:
Of course this PR would be made right as I realize that there's a lot of handy things that can be grabbed from the science outpost for use in normal science, such as a heater, hydroponics tray, additional tank storage unit, and space cleaner. But then again I suppose that does prove the issue with the science outpost: _its useful for everything except for its intended purpose_.

The inability to go grab a heater/hydro tray from there's a bit annoying, but that can be replaced. The things that can be grabbed from there that will truly be missed due to their inability to really be replaced is the extra tank storage unit (for toxins) and the space cleaner (multi-faceted use _cough_). Another thing I really liked the science outpost was the fact that it was somewhere to evacuate the station from in the case of a nuke that was on the Starboard side of the station, as opposed to the mining shuttle, or in some cases, the engineering shuttle on the Port side.

Fox-McCloud commented:
@FreeStylaLT 

You could probably just use meta-station's map for both Cyberiad and Meta.

Rumiluntti commented:
>Would there be a possibility of moving the anomaly research stuff on to station z-level? Could use the free space from where the shuttle was to hold all of it. I would love to actually RESEARCH anomaly cores instead of robotics just hoovering them up for meme mechs. Usually though its to much of a bother and risk to head to the outpost when an anomaly shows up do to it usually being an antag haven.

Anomaly research is literally just sticking it in the DA and checking if it's any good.
The machines in the ooutpost do jack shit for the event anomalies, they are for xenoarch anomalies and xenoarch was removed ages ago.

ktoma36 commented:
Awww, oh well. So much for being a full on anomalist

Ar3nn commented:
#bringbackxenoarch2017





No? Ok.

Fox-McCloud commented:
Ya messed up.

SamHPurp commented:
Really you should rebase and add your bits over.

FreeStylaLT commented:
Fixed the conflicts, made z6 and z7 be used by both maps, removed individual z5s and made the new Cyb's z5 a generic one.

I also think I forgot to remove cyberiad's z6 and z7. I should fix that.

SamHPurp commented:
All the generic stuff you're doing I've literally already done in a PR. #7396

FreeStylaLT commented:
well fiddle me three ways 'till sunday this is a right mess

SamHPurp commented:
@FreeStylaLT this is what I was trying to say on Discord yesterday, but I guess we both got our wires crossed.

Fox-McCloud commented on _maps/cyberiad.dm:
```diff
@@ -18,9 +18,9 @@ z7 = empty
-        #include "map_files\cyberiad\z6.dmm"
-        #include "map_files\cyberiad\z7.dmm"
+        #include "map_files\generic\z5.dmm"
+        #include "map_files\generic\z6.dmm"
```
This one isn't ready to be ripped out yet; it has the Russian direlict, Ruski DJ, and syndicate outpost.

Fox-McCloud commented on _maps/cyberiad.dm:
```diff
@@ -18,9 +18,9 @@ z7 = empty
         #include "map_files\cyberiad\z3.dmm"
         #include "map_files\cyberiad\z4.dmm"
-        #include "map_files\cyberiad\z5.dmm"
-        #include "map_files\cyberiad\z6.dmm"
```
This one isn't ready to be ripped out yet; it has the Russian direlict, Ruski DJ, and syndicate outpost.

Fox-McCloud commented on _maps/cyberiad.dm:
```diff
@@ -18,9 +18,9 @@ z7 = empty
         #include "map_files\cyberiad\z4.dmm"
-        #include "map_files\cyberiad\z5.dmm"
-        #include "map_files\cyberiad\z6.dmm"
-        #include "map_files\cyberiad\z7.dmm"
```
Delete this one from existence.

Fox-McCloud commented on _maps/cyberiad.dm:
```diff
@@ -18,9 +18,8 @@ z7 = empty
-        #include "map_files\cyberiad\z5.dmm"
+        #include "map_files\generic\z5.dmm"
         #include "map_files\cyberiad\z6.dmm"
-        #include "map_files\cyberiad\z7.dmm"
```
Urf, I'm an idiot. you misunderstood what I meant.

Get rid of cyberiad\z7.dmm the *map*, but leave it in, as a generic.

Fox-McCloud replied:
this should be `#include "map_files\generic\z7.dmm"`

SamHPurp commented:
Foooooox - I literally already had a PR up for ages now that does all this (other than the Sci outpost thing)....

Not had any feedback, and now its being tacked onto another PR?...

SamHPurp commented:
If you're going to rip my standardisation PR can you at least get all the bits that matter too? Like the `DECLARE_LEVEL`s to be the same?...

Fox-McCloud commented:
You standardized *all* z-levels though. I know @tigercat2000 took issue with that.

Considering z-level 5 of meta and cyberiad are practically identical, now, though, and the fact that 7, on both, *are* identical...well, that's why these were done.

SamHPurp commented:
@Fox-McCloud Uhh no. I don't touch 2 or 3. (Although would love to).

You say 5 is identical? That means you really haven't looked at this, because they aren't identical in the slightest. Meta and Box's 4 is closer to identical than 5.

This is fucking mental. 

2 months with no feedback on my near identical PR. And Frees get merged.

KasparoVy commented:
Feels bad, man

Big problems

matt81093 commented:
It was merged before Declare level could be added, I assume. Which is sad. Linky your PR? @SamHPurp

Aurorablade commented:
why are you necroposting on a month old PR besides being a troll?
